original_sdk/client.py

```diff
@@ -160,3 +160,24 @@ def get_burns_by_user_uid(self, user_uid: str) -> OriginalResponse:
 
     def get_deposit(self, user_uid: str) -> OriginalResponse:
         return self.get("deposit", params={"user_uid": user_uid})
+
+    def get_reward(self, reward_uid: str) -> OriginalResponse:
```

Comment:
In the backend is this a "get" or a "list"?

When we get via a "primary key" (i.e. the database object via uid) we use "uid" as the param name. Otherwise if its a search/query, we use the named parameter "reward_uid".